lblod/ember-rdfa-editor
# GN-4614: automatically add http or mailto to links

### Overview
Uses Linkify.js to match links and add http or mailto when relevant. Also fixes a few usability bugs around links:
- Fix a crash after reloading the page and using the sidebar
- Some missing translations
- Selection when inserting a link (so hitting insert and then typing writes the text of the link instead of replacing the new link node with the character typed)

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4614

### Setup
N/A

### How to test/reproduce
Type a valid or invalid link (or email) into either of the href boxes (in pop-over or sidebar) and see that the http: or mailto: part is added automatically.

### Challenges/uncertainties
There are more usability problems I wanted to fix, but I wasn't able to figure out how to do them as embedded-editor nodes seem to have problems around reactivity in the ember components (e.g. changing a property with a `@tracked` decorator not updating in the UI).

### Checks PR readiness
